### PR TITLE
community.aws 2.6.0 release prep

### PR DESCRIPTION
**Pull request by mandar242**
##### SUMMARY

community.aws 2.6.0 release

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request

##### COMPONENT NAME

community.aws collection

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,191 +29,191 @@ Version 2.0.0 of this collection supports `boto3 >= 1.15.0` and `botocore >= 1.1
 ### Connection plugins
 Name | Description
 --- | ---
-[community.aws.aws_ssm](https://github.com/ansible-collections/community.aws/blob/stable-2/docs/community.aws.aws_ssm_connection.rst)|execute via AWS Systems Manager
```
Comment (markuman) [inline README.md:32]: `collection_prep_add_docs` must be run against branch `stable-2`.

`collection_prep_add_docs -b stable-2 -p .`

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,24 @@ community.aws Release Notes
...
+- ecs_service_info - add ``name`` alias to address the ecs service name with the same parameter as the ecs_service module is doing (https://github.com/ansible-collections/community.aws/pull/1187).
+
+Bugfixes
+--------
+
+- ecs_service - fix broken change detect of ``health_check_grace_period_seconds`` parameter when not specified (https://github.com/ansible-collections/community.aws/pull/1212).
+- ecs_service - use default cluster name of ``default`` when not input (https://github.com/ansible-collections/community.aws/pull/1212).
+- ecs_task - dont require ``cluster`` and use name of ``default`` when not input (https://github.com/ansible-collections/community.aws/pull/1212).
+- wafv2_ip_set - fix bug where incorrect changed state was returned when only changing the description (https://github.com/ansible-collections/community.aws/pull/1211).
+
```
Comment (markuman) [inline CHANGELOG.rst:25]: Maybe add also a `release_summary` similar like that

```yml
release_summary: |-
  This is the last planned stable-2 release of ``community.aws`` collection.  
  Consider to upgrade to stable-3 soon.
```

https://docs.ansible.com/ansible/devel/community/collection_contributors/collection_release_without_branches.html#id4

Comment (mandar242) [reply CHANGELOG.rst]: added.

File: README.md (hunk 1 above)
Comment (mandar242) [reply README.md]: oh right! fixed.

File: docs/community.aws.aws_batch_job_definition_module.rst [hunk 3]
```diff
@@ -28,8 +28,8 @@ Requirements
 The below requirements are needed on the host that executes this module.
 
 - python >= 3.6
-- boto3 >= 1.15.0
-- botocore >= 1.18.0
+- boto3 >= 1.17.0
+- botocore >= 1.20.0
```
Comment (markuman) [inline docs/community.aws.aws_batch_job_definition_module.rst:32]: This must not change/increase in the stable-2 release.  
I guess you must removeyour local amazon.aws >= 3.0.0 collection and install the latest stable-2 release before generating the docs.

Comment (mandar242) [reply docs/community.aws.aws_batch_job_definition_module.rst]: understood, pushed the updated code.
I switched to [amazon.aws/stable-2](https://github.com/ansible-collections/amazon.aws/tree/stable-2) locally, then ran the `add_docs`, does that sound correct?

**Review by markuman — APPROVED** [review]

**Review by alinabuzachis — APPROVED** [review]

File: CHANGELOG.rst [hunk 4]
```diff
@@ -5,6 +5,30 @@ community.aws Release Notes
...
 
 
+v2.6.0
+======
+
+Release Summary
+---------------
+
+This is the last planned stable-2 release of ``community.aws`` collection.
+Consider to upgrade to stable-3 soon.
```
Comment (tremble) [inline CHANGELOG.rst:15]:
```suggestion
This is the last planned 2.x release of the ``community.aws`` collection.
Consider upgrading to the latest version of ``community.aws`` soon.
```

Comment (mandar242) [reply CHANGELOG.rst:15]: Fixed.

**Pull request closed without merging**